### PR TITLE
feat: ssr safe rendering

### PR DESCRIPTION
**Pull request by DorianMaliszewski**
## Type

- Enhancement

#### What is expected?

- Scaleway can work with SSR

#### The following changes where made:

1. Based on SCW Website and thanks to `yalc`, I successfully add the library to the website.

2. Change when needed call to Client API instead of Node API

### CODE REVIEW

**Review by matthprost — APPROVED** [review]
🎉

File: src/components/Toaster/index.js [hunk 1]
```diff
@@ -5,11 +5,9 @@ import {
   ToastContainer as BaseToastContainer,
   toast as baseToast,
 } from 'react-toastify'
-import { injectStyle } from 'react-toastify/dist/inject-style'
 import Icon from '../Icon'
 import NotificationBar from '../NotificationBar'
-
-injectStyle()
+import 'react-toastify/dist/ReactToastify.css'
```
Comment (chambo-e) [inline src/components/Toaster/index.js:10]: Does this correctly inline the css in the bundle ?

Comment (philibea) [inline src/components/Toaster/index.js:10]: I got a doubt on this change, i think we use to be like this before but it's doesn't work properly on other projects

Comment (DorianMaliszewski) [reply src/components/Toaster/index.js:10]: Do I need to do the same as you did with PhoneInput css ?

Comment (chambo-e) [reply src/components/Toaster/index.js:10]: If this is not inlined yes I think you'll need to

But what's wrong with the injectStyle way ?

Comment (DorianMaliszewski) [reply src/components/Toaster/index.js:10]: injectStyles use document object under the hood which is not supported in SSR/SSG mode. The other way to keep it is to put injectStyles() in useLayoutEffect/useEffect on mount lifecycle

Comment (DorianMaliszewski) [reply src/components/Toaster/index.js:10]: injectStyle use document object under the hook (See https://github.com/fkhadra/style2js/blob/main/src/style2js.ts). NextJS doesn't like it 😅

Comment (chambo-e) [reply src/components/Toaster/index.js]: I just checked and css seems to be correctly inlined in the bundle, so no issue on my side :)

**Review by chambo-e — APPROVED** [review]

**Review by matthprost — APPROVED** [review]

**Pull request merged**